tests/test_database_update.py

```diff
@@ -165,3 +166,26 @@ def run_and_test_bowtie2_on_updated_db(
     )
     filter_output.filter_sam(output_file_path, filtered_output_file_path)
     assert hash.unordered(filtered_output_file_path) == expected_hash
+
+
+def download_db_files_for_local_tests(input_dir_path):
+    db_file_names = [
+        "BDGP6.1.bt2",
+        "BDGP6.2.bt2",
+        "BDGP6.3.bt2",
+        "BDGP6.4.bt2",
+        "BDGP6.rev.1.bt2",
+        "BDGP6.rev.2.bt2",
+    ]
+
+    os.makedirs(input_dir_path, exist_ok=True)
+    downloaded_full_paths = []
+    for db_file_name in db_file_names:
+        downloaded_full_path = os.path.join(input_dir_path, db_file_name)
+        if not os.path.isfile(downloaded_full_path):
+            s3.download_integration_test_input(
+                s3_file_key=f"databases/bowtie2-fruitfly-BDGP6/{db_file_name}",
+                output_file_path=downloaded_full_path,
+            )
+        downloaded_full_paths.append(downloaded_full_path)
+    return downloaded_full_paths
```

Comment:
Note that this database does not have any subdirectories. We currently don't have a database that does all of the following:
- Has subdirectories that are actually used by the tool
- Is small enough to download locally and re-upload without taking a significant amount of time

We can add a dummy subdirectory+subfile to test functionality, but this integration test wouldn't be able to detect automatically that the subfile was transferred.